common/src/lazy_pages.rs

```diff
@@ -23,48 +23,47 @@ use gear_core::{
     ids::ProgramId,
     memory::{HostPointer, Memory, PageBuf, PageNumber},
 };
-use gear_runtime_interface::{gear_ri, GetReleasedPageError, MprotectError};
+use gear_runtime_interface::{gear_ri, RIError};
 use sp_std::{
     collections::{btree_map::BTreeMap, btree_set::BTreeSet},
     vec::Vec,
 };
 
 #[derive(Debug, Clone, PartialEq, Eq, derive_more::Display)]
 pub enum Error {
-    #[display(fmt = "{}", _0)]
-    Mprotect(MprotectError),
-    #[display(fmt = "{}", _0)]
-    GetReleasedPage(GetReleasedPageError),
-    #[display(fmt = "Cannot convert vec to page data")]
-    VecToPageData,
-    #[display(fmt = "Cannot find page data in storage")]
-    NoPageDataInStorage,
-    #[display(fmt = "Released page {:?} has initial data", _0)]
-    ReleasedPageHasData(PageNumber),
+    #[display(fmt = "RUNTIME INTERFACE ERROR: {}", _0)]
```

Comment:
Why caps?

Reply:
Because this is not common error - this error can lead to problems in consensus or UB.